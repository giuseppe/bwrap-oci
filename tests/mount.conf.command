#!/bin/sh
oci-runtime-tool generate --bind=foo:bar
