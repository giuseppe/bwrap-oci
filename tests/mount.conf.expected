bwrap --as-pid-1 --die-with-parent --bind rootfs / --unshare-cgroup --unshare-pid --unshare-net --unshare-ipc --unshare-uts --unshare-user --cap-drop ALL --cap-add CAP_AUDIT_WRITE --cap-add CAP_NET_RAW --cap-add CAP_DAC_OVERRIDE --cap-add CAP_SETPCAP --cap-add CAP_FSETID --cap-add CAP_FOWNER --cap-add CAP_SETUID --cap-add CAP_MKNOD --cap-add CAP_CHOWN --cap-add CAP_NET_BIND_SERVICE --cap-add CAP_KILL --cap-add CAP_SETGID --cap-add CAP_SETFCAP --cap-add CAP_SYS_CHROOT --chdir / --setenv PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin --setenv TERM xterm --uid 0 --gid 0 --proc /proc --dev /dev --bind /dev/pts /dev/pts --tmpfs /dev/shm --mqueue /dev/mqueue --ro-bind /sys /sys --bind foo bar --tmpfs /tmp --hostname mrsdalloway --block-fd FD --sync-fd FD --info-fd FD --seccomp FD sh
